### DISCUSSION TITLE
Samsung TV

### DISCUSSION THREAD

**Discussion by TriderG75** [RavenSystem/esp-homekit-devices#1822] [IR Codes Collection]
- Vendor: Samsung 

- Type of device: TV

- Remote model: AA59-00622A

- Frequency (KHz): 38KHz (Default)

- Protocol: `J(J(AcAcAcDFAc`

- HAA Codes: 

| Function | HAA Code |
|------|------|
| **On/Off**|              CeCfAfAaF 
| **On**       | 		        CeCeAbBbAaBbBa 
|**Off**		|        CeChBbDbBa
|**CH+**		|CeCfAbAcAaBaC
|**CH-**		 |       CeCiAcDaC
|**V+**		  |      CeCeChE	
|**V-**		   |     CeCeBaAfAaD 
|**Menu**	|	CeCfAaBcAaAbC 
|**Mute**	|	CeCeDhD
|**Up**		 |       CeCjBaEbA
|**Down**	|	CeCeAdBbDbA
|**Left**		 |       CeCeAaAbBbAaBbA
|**Right**	|	CeCfAcBaAaCbA
|**Ok**		|CeChAaBaCaAbA
|**Back**|		CeChBaAaCbAaA
|**Exit**	|	CeCeAaBaAcAbAaB
|**Source**|		CeCeAhG
|**Tools**	|	CeCeBaAbAcAaBaA
|**Content**	|	CeCeAbDbBdA
|**CH List**	|CeCeBaAaBcAaAbA
|**Play**		|CeCeCcAdCaA
|**Pause**	|	CeCfAaAbAaAaAaBaA
|**Info**		|CeCeEhC
|**Hdmi**	|	CeCeBaAcAbAaCa
|**Hdmi1**	|	CeCeAbAaCaBaAc
|**Hdmi2**	|	CeCfEaBeAa
|**Hdmi3**	|	CeCfAdCaDb
|**Hdmi4**	|	CeCeAaAcBaAaCb		
|**Stop**	|	CeCfBcAaAbCaA
|**1**		|CeCgAeBaECeCgAeBaE
|**2**		|CeCeAaAfAaE
|**3**		|CeCfBeAbE
|**4**		|CeChAdCaD
|**5**		|CeCeAbAeBaD
|**6**		|CeCfAaAdAaAaD
|**7**		|CeCgBdBbD
**8**		|CeCeAaBeAbD
**9**		|CeCfCdAcD
**0**		|CeCeAcAdCaC

**Comment by manur84** [comment]
Vendor: Samsung UE34F4

    Type of device: TV

    Remote model: AA59-00786A

    Frequency (KHz): 38KHz (Default)

    Protocol: J8J(AQDHAaDHAaAgAQAgAg

    HAA Codes:

- >  power
- > aAaoOnAaANoOnANoOnANAaAc
- > 
- > channelup
- > aBnOoaAaoOpNoaANaAaBnNa
- > 
- > channelup
- > aAanNpAaAoNoaNoNAaOaAaAOA
- > 
- > left
- > aAanPncNnPANaObNnANAaoa
- > 
- > right
- > aBoOnAaANoOnANoaANanaAaoa
- > 
- > MENU
- > aAaoOnAbNoOnANAaoNanAnNC
- > 
- > mute
- > aAanPncNnPAaAanPnPAaAa
- > 
- > source
- > aBnNoNaAaoNoApNpNFa
- > 
- > ok
- > aAanPncNoOoNaNaANaAanaoa
- > 
- > back
- > cCoOanAnAnCNanaANaNBNna
- > 
- > play
- > aAanNoNaAaoNoAaANnNaoOAaANa
- > 
- > info
- > aAaoOnAbNoOAaAaAOoOoaAa
- > 
- > volume+
- > aAanOnNaAaoOnAaANnPnOaAaAa
- > 
- > volume-
- > aAanPncNnPAanaNnPnaNAaAa
- > 
- > show tv setting
- > aAaoNoAaANoOAanANnANoaNAana
-
